routers/init.go

```diff
@@ -15,6 +15,7 @@ import (
 	"code.gitea.io/gitea/modules/cache"
 	"code.gitea.io/gitea/modules/cron"
 	"code.gitea.io/gitea/modules/highlight"
+	"code.gitea.io/gitea/modules/indexer/issues"
```

Comment:
you should probably also use `issue_indexer` here to be consistent in all places

Reply:
done.